### PR TITLE
Fix links and remove overwriting CSS

### PR DESCRIPTION
**Pull request by pamo**
This is a big PR but the typography lives in one place now.

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -85,52 +85,55 @@ h3,
 h4,
 h5,
 h6 {
-  color: $primary-font-color-dark;
+  color: $heading-font-color;
   font-family: $heading-font;
   font-weight: $heading-font-weight;
-  line-height: line-height(2);
   margin: line-height(2) 0 0 line-height(-1);
```
Comment (lizgene) [inline _sass/_base.scss]: While we're at it, can we get rid of this right margin on the header elements?

image

File: _posts/2017-03-19-audre-lorde-sister-outsider-study-guide.md [hunk 2]
```diff
@@ -8,338 +8,139 @@ tags: feminism anti-racism anti-capitalism
...
 
 # Sister Outsider
-by Audre Lorde
+## by Audre Lorde
 
-## Introduction question
+### Introduction Question
 
--   Which essay stood out to you (positively or negatively) and why?
+* Which essay stood out to you (positively or negatively) and why?
```
Comment (lizgene) [inline _posts/2017-03-19-audre-lorde-sister-outsider-study-guide.md]: And let's kill this bullet point, since it's not actually part of any list.

**Review by lizgene — CHANGES_REQUESTED** [review]

File: _sass/_base.scss (hunk 1 above)
Comment (lizgene) [reply _sass/_base.scss]: And by `right` I mean `left`

**Comment by pamo** [comment]
Done @lizhubertz

**Comment by lizgene** [comment]
LGTM! 👍

**Review by lizgene — APPROVED** [review]

**Pull request merged**